examples/MagicWeather/build.gradle

```diff
@@ -2,15 +2,15 @@
 buildscript {
     ext.compileVersion = 31
     ext.kotlinVersion = "1.6.21"
-    ext.purchasesVersion = "6.0.0-alpha.4"
+    ext.purchasesVersion = "6.1.1"
```

Comment:
I don't know why we were pointing to the alpha version here but I was running into some issues so instead of trying to fix things in that version, I just updated to the latest (and had to do a couple of changes as a result)